<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>614</height>
   </rect>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>71</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <widget class="QPushButton" name="btn_LoadLibrary">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>96</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Library</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_ShowNeurons">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_ThresInnervLength">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Inn. thres:</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="ckbx_IncludeGlomeruli">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>40</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Include glomeruli</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_DataSource">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>10</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Data source</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTableViewWithComboFilter" name="NeuronTable"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTableViewWithComboFilter</class>
   <extends>QTableWidget</extends>
   <header>QTableViewWithComboFilter</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
